src/components/layout.js

```diff
@@ -27,7 +28,14 @@ const Layout = ({ location, title, children }) => {
   return (
     <div>
       <header id="site-header-main">
+        <link
+          rel="stylesheet"
+          href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+        />
```

Comment:
We already have Font Awesome in use via https://github.com/FortAwesome/react-fontawesome. In `layout.js`, you can declare what you need and then reference them inline.